### PR TITLE
27 list devices

### PR DESCRIPTION
**Pull request by majorz**

### CODE REVIEW

File: src/connection/mod.rs [hunk 1]
```diff
@@ -145,16 +145,16 @@ pub fn disable(connection: &mut Connection, time_out: i32) -> Result<(), String>
...
+                                                                     .into())]);
             dbus_connect!(message);
 
             wait(connection, time_out, ConnectionState::Deactivated)
         }
     }
 }
 
-#[test]
+//#[test]
```
Comment (josephroberts) [inline src/connection/mod.rs]: Can you comment this back in and try to figure out why it fails 50% of the time?
You will need to `export  TEST_WIFI_SSID=resin_io` first, replacing `resin_io` with your SSID.

File: src/dbus_helper/mod.rs [hunk 2]
```diff
@@ -25,3 +30,77 @@ macro_rules! dbus_connect {
             .unwrap_or_else(|e| panic!("dbus_connect error: {}", e))
     }}
 }
+
+#[inline]
```
Comment (josephroberts) [inline src/dbus_helper/mod.rs]: Why have you used `inline` for most functions? The rust reference says "The compiler automatically inlines functions based on internal heuristics. Incorrectly inlining functions can actually make the program slower, so it should be used with care"

Comment (majorz) [inline src/dbus_helper/mod.rs]: This is indeed so. I will remove all `inline` declarations from my code.

File: src/connection/mod.rs (hunk 1 above)
Comment (majorz) [inline src/connection/mod.rs]: Working on figuring it out now.

**Pull request merged**